graphit_eval/devcloud_eval_dataset2/table7/benchmark.py

```diff
@@ -48,15 +48,15 @@
                                   "sssp" : "sssp_hybrid_denseforward",
                                   "cc" : "cc_hybrid_dense_bitvec_segment",
                                   "bfs" :"bfs_hybrid_dense_bitvec",
-                                  "bc" : "bc_SparsePushDensePull_bitvector",
+                                  "bc" : "bc_SparsePushDensePull_bitvector_openmp",
```

Comment:
maybe name the other binary as "bc_SparsePushDensePull_bitvector_cilk"? Just to differentiate between the 2?

Reply:
yep makes sense. will update it